src/components/calcite-block/calcite-block.tsx

```diff
@@ -189,13 +189,18 @@ export class CalciteBlock {
       </div>
     );
 
+    const slotNode = <slot />;
+
     return (
       <Host>
         <article aria-expanded={collapsible ? (open ? "true" : "false") : null} aria-busy={loading}>
           {headerNode}
           <div class={CSS.content} hidden={!hasContent || !open}>
-            <slot />
-            {loading || disabled ? <CalciteScrim loading={false}></CalciteScrim> : null}
+            {loading || disabled ? (
+              <CalciteScrim loading={false}>{slotNode}</CalciteScrim>
```

Comment:
@jcfranco I could always put the slot inside the scrim since the loader is only shown when its loading but then it would always have a position:relative and zIndex of 1 for the container. That might be ok though.